app/views/developers/_details.html.erb

```diff
@@ -18,4 +18,6 @@
       </div>
     </div>
   </div>
+
+  <%= render TimeZoneComponent.new(developer.time_zone) %>
```

Comment:
If you add a time zone but no social links you get two lines.

<img width="342" alt="image" src="https://user-images.githubusercontent.com/2092156/145256910-8011524b-262c-4cdc-97ba-2c402658a19e.png">

Let's use [divide](https://tailwindcss.com/docs/divide-color) and remove the individual borders. That will be more future proof in case we add another section/area, too.

Reply:
I think this should be taken care of in `main` now.

Reply:
good deal, I'll grab the latest and rebase.